javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1654,9 +1654,10 @@ class TargetLocator {
    *     when the driver has changed focus to the specified window.
    */
   window(nameOrHandle) {
+    let paramName = this.driver_.getExecutor().w3c ? 'handle' : 'name';
```

Comment:
I'd rather just send the parameter twice than break encapsulation here. There's already precedence with webelement IDs